siac/walletcmd.go

```diff
@@ -207,7 +211,10 @@ func walletload033xcmd(source string) {
 
 // walletloadseedcmd adds a seed to the wallet's list of seeds
 func walletloadseedcmd() {
-	password, err := speakeasy.Ask("Wallet password: ")
+	fmt.Println("Loading seed...")
+	time.Sleep(time.Second * 3)
+	fmt.Println("Load successful.  Encrypting with current encryption password")
+	password, err := speakeasy.Ask("Need Password: ")
```

Comment:
this prints "load successful" before it even asks you for the thing it's loading!

![crazy pills](https://media.giphy.com/media/NPyHgTkMStCXC/giphy.gif)